pkgs/leak_tracker_flutter_testing/lib/src/leak_tracking_for_tests.dart

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:leak_tracker/leak_tracker.dart';
+import 'package:meta/meta.dart';
+
+void _emptyLeakHandler(Leaks leaks) {}
+
+/// Leak tracking settings and helper methods for tests.
+abstract class LeakTrackingForTests {
```

Comment:
It is not entirely clear to me how this class is supposed to be used (some docs (incl. usage examples)) would help).

in the example given in the comment it appears to be used to configure a `testWidgets` test case - I am confused that it contains mutable state, though. Can you change the test configuration while a test is running? Why is that necessary?

Reply:
Added comment for the class.